.github/workflows/push.yml

```diff
@@ -0,0 +1,95 @@
+name: Entur/Docker/Push
+
+on:
+  workflow_call:
+    inputs:
+      image_artifact:
+        description: "GitHub artifact with Docker image"
+        type: string
+        default: "repo_name"
+      artifact_registry_project_id:
+        type: string
+        description: "Project id for artifact registry"
+        default: "entur-system-1287"
```

Comment:
I'm not entirely sure that we should allow setting this?

Reply:
I agree.